RemoteEngine/docker/README.md

```diff
@@ -81,12 +81,19 @@ While starting a remote engine, following optional flags can be used in addition
 
 
 ## Upgrade
+
 Upgrade the remote engine instance to the latest version.
+* Note that the `./dsengine.sh cleanup` command will cleanup the persistent storage directories, so its not recommended for an upgrade.
+* The `./dsengine.sh startup` command will check if there is an existing container with the name specified, and if there is, the same container will be started.
```

Comment:
Doesn't the startup abort if the container with same name exists?

Reply:
Nope, as explained above: 
> The `./dsengine.sh startup` command will check if there is an existing container with the name specified, and if there is, the same container will be started.

Let me know if this can be phrased in a simpler way!